Minitwit_BE/Minitwit_BE.Domain/User.cs

```diff
@@ -1,10 +1,47 @@
-﻿namespace Minitwit_BE.Domain
+﻿using System.Security.Cryptography;
```

Comment:
From the security point of view, it's not recommended to send passwords in plain text through HTTP requests. 

What I would suggest would be to hash a password in the frontend, send a hashed password and only compare (or insert) the hashes in the BE.

Reply:
true, I was following the python equivalent from the lecture too much